CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,14 @@
+# Observatorium Community Code of Conduct
+
+Observatorium follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Additionally, our team members can be contacted directly for code of conduct abuse, as described below.
```

Comment:
Just to clarify that it's not to contact us in order to abuse the code of conduct

```suggestion
Observatorium follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Additionally, our team members can be contacted directly concerning code of conduct abuse, as described below.
```